context/app/static/js/components/Detail/Protocol/Protocol.jsx

```diff
@@ -26,12 +27,18 @@ function ProtocolLink(props) {
 function Protocol(props) {
   const { protocol_url } = props;
 
+  const matchedDoiSuffix = protocol_url.match(/\w+$/)[0];
+
+  const protocolData = useProtocolData(matchedDoiSuffix, 1);
+
+  const doi = 'protocol' in protocolData && 'doi' in protocolData.protocol ? protocolData.protocol.doi : '';
```

Comment:
So, frustratingly, I'm guessing that the thing they call `doi` here isn't really a DOI. 
```suggestion
  const resolverHostnameAndDOI = 'protocol' in protocolData && 'doi' in protocolData.protocol ? protocolData.protocol.doi : '';
```